src/NeonView.js

```diff
@@ -69,7 +69,7 @@ class NeonView {
   /**
    * Get the current page from the loaded view and then display the
    * most up to date SVG.
-   * @param {boolean} delay. whether or not to delay loading the page by 500ms. defaults to false
+   * @param {boolean} [delay=false]. whether or not to delay loading the page by 500ms. defaults to false
```

Comment:
The period is not correct syntax. Change to `[delay=false] - whether`. Currently JSDoc would throw a warning and interpret the variable as being named `delay.`.